### PR TITLE
Added several implementations in sceRtc.

### PR DESCRIPTION
**Pull request by Rinnegatamante**
This implements:

- sceRtcCheckValid
- sceRtcGetDayOfWeek
- sceRtcGetDaysInMonth
- sceRtcIsLeapYear
-  sceRtcTickAddDays
- sceRtcTickAddHours
- sceRtcTickAddMicroseconds
- sceRtcTickAddMinutes
- sceRtcTickAddSeconds
- sceRtcTickAddTicks
- sceRtcTickAddWeeks

### CODE REVIEW

**Comment by Zangetsu38** [comment]
broken Quake for me :/

**Comment by VelocityRa** [comment]
@Zangetsu38 My new PR will fix it

File: src/emulator/modules/SceRtc/src/SceRtcUser.cpp [hunk 1]
```diff
@@ -21,8 +21,57 @@
 
 #include <ctime>
 
-EXPORT(int, sceRtcCheckValid) {
-    return unimplemented("sceRtcCheckValid");
+#define VITA_CLOCKS_PER_SEC 1000000
```
Comment (petmac) [inline src/emulator/modules/SceRtc/src/SceRtcUser.cpp]: This will end up being a redefinition of the macro when #61 is merged

**Comment by Rinnegatamante** [comment]
@petmac any idea why build fails here?

**Comment by petmac** [comment]
I think because the vitasdk power.h changed to an older version.

**Comment by Rinnegatamante** [comment]
Should be ready to merge now.

**Pull request merged**